spec/requests/api/v1/countries/index_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe 'GET /api/v1/countries', type: :request do
+  subject { get api_v1_countries_path, headers: headers, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+
+  context 'when there are countries created' do
+    let(:countries) { create_list(:country, 2) }
+
+    before do
```

Comment:
We can avoid the `before` block by adding a `!` to the `let` and `subject`
- https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject#use-the-%60subject!%60-bang-method-to-call-the-definition-block-before-the-example
- https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let#use-let!-to-define-a-memoized-helper-method-that-is-called-in-a-before-hook

Reply:
I prefer that many of these `let` not be loaded into memory for performance reasons but I have updated these changes as requested.

Reply:
Sure! I was just giving my point of view and sharing some links with you.
Feel free to not apply suggestions if you are not comfortable with them 😊